test/helpers/browser.js

```diff
@@ -0,0 +1,5 @@
+var jsdom = require('jsdom');
+const {JSDOM} = jsdom;
+
+const {document} = (new JSDOM('')).window;
+global.document = document;
```

Comment:
ℹ️ This may have been one of the issues preventing tests from running in the first try, but with this change, JSDOM no longer puts a huge warning into the console whenever tests are ran.